go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/casdoor/go-sms-sender
 
-go 1.17
+go 1.21
```

Comment:
Don't change this

Reply:
Oh, sorry, I accidentally touched this line during testing